modules/media/js/editForm.js

```diff
@@ -98,6 +96,8 @@ var MediaEditForm = React.createClass({
       }
     }
 
+    var backURL = loris.BaseURL.concat('/media/');
```

Comment:
@alisterdev, I am going to close this PR but the backURL thing should get in 17.0